__std__/nix/doc/scripts/default.nix

```diff
@@ -4,5 +4,5 @@
 
   sphinx-autobuild-doc-site = import ./sphinx-autobuild-doc-site.nix organelle;
```

Comment:
Change name to `sphinx-autobuild-readthedocs-site`?

Reply:
Yes 👍